src/Files.App/UserControls/InnerNavigationToolbar.xaml

```diff
@@ -696,7 +696,7 @@
 									Padding="0"
 									HorizontalContentAlignment="Center"
 									VerticalContentAlignment="Center"
-									AutomationProperties.Name="{helpers:ResourceString Name=LargeIcons}"
+									AutomationProperties.Name="{helpers:ResourceString Name=NavToolbarLargeIcons/AutomationProperties/Name}"
```

Comment:
This change looks wrong 🤔

Reply:
I don't remember having done that, must have been a merge or something, I'll revert now